Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -0,0 +1,172 @@
+# Detailed overview of ArangoSearch views
+
+## What is ArangoSearch view
+
+### ArangoSearch is much more than a fulltext search
+
+But fulltext searching is a subset of its available functionality, supported via
+the 'text' analyzer and 'tfidf'/'bm25' [scorers](Scorers.md), without impact to performance
+when specifying documents from different collections or filtering on multiple
+document attributes.
+
```

Comment:
Would ditch _What is ArangoSearch view_ headline completely and either remove _ArangoSearch is much more than a fulltext search_ or use it as regular sentence, but not as headline (marketing style should be avoided in docs). Also, the first paragraph is strange:

> But fulltext searching is a subset of its available functionality

It doesn't seem to fit together with the headline, especially the "But" is odd (has the notion of a limitation, although we want to advertise its merits).